src/codegen.ts

```diff
@@ -144,6 +148,30 @@ export async function ${funcName}(
     const result = await sql.unsafe(\`${sql}\`${queryValues})
     return ${outputValue}
 }
+`
+  },
+  'pg-promise'({
+    sourceFileName,
+    module,
+    funcName,
+    params,
+    returnType,
+    sql,
+    queryValues,
+    outputValue,
+  }: GeneratorOptions) {
+    return `\
+${topComment(sourceFileName)}
+
+import pgp from '${module}'
```

Comment:
One more nitpick: This should be `import * as pgp from '${module}'`, because otherwise the users are required to have `"esModuleInterop": true` in their `tsconfig.json`.